Makefile

```diff
@@ -0,0 +1,40 @@
+HOST = localhost:4000
+NPM_BIN = ./node_modules/.bin
+export PATH := $(NPM_BIN):$(PATH)
+
+# Install dependencies
+install:
+	@echo "Installing Pa11y..."
+	@npm install pa11y@^3
+	@echo "Installing GitHub Pages gem..."
+	@gem install github-pages
+
+# Build the site
+build:
```

Comment:
I can't really take credit for those, I _stole_ from Rowan's version on [pa11y.github.io](https://github.com/pa11y/pa11y.github.io/blob/master/Makefile).